web/next.config.js

```diff
@@ -4,7 +4,8 @@
 // @ts-ignore
 const nextSafe = require("next-safe");
 const isDev = process.env.NODE_ENV !== "production";
-
+const s3BucketUrl = `https://s3.${process.env.ASSETS_S3_REGION}.amazonaws.com/${process.env.ASSETS_S3_BUCKET_NAME}`;
+const cdnUrl = process.env.ASSETS_CDN_URL;
```

Comment:
@pdtfh It is unclear to me if we need to have region name in prod, but on my local dev I did need it.

Reply:
This does not work with the presigned url domain returned by presignedpost